### ISSUE TITLE
Chore: Remove empty / default function / property declarations

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/recipes-flux#15]
We've got a ton of these:

```js
  displayName: "Home",
  propTypes: {},
  mixins: [],
  getInitialState: function () { return null; },
  componentWillMount: function () {},
  componentWillUnmount: function () {},
```

Which are unnecessary and generally a bad practice. We should scour and remove everything that is a default / noop value and only add properties / functions for things we actually implement.

**Comment by ryan-roemer** [comment]
@colinmegill -- Let's see if we can't get this cleaned up before students actually start using the repo (I'm guessing this boilerplate is here to slightly speed you up when you don't know what you're overriding in initial dev...)

**Comment by colinmegill** [comment]
@ryan-roemer yeah, you guessed it - but the no ops are there intentionally for educational reasons so that we can add things to lifecycle methods. There is no broken behavior, removing bug label.

**Comment by ryan-roemer** [comment]
@colinmegill -- OK. Makes sense. Maybe we could scope to **only** the files the presentation will actually edit / enhance (if not already scoped that way)?

Removing `wontfix`. Please close issue when properly scoped (if not already).

**Comment by colinmegill** [comment]
That's a great idea - even better would be to scope to a branch that that
task will be accomplished on.
